src/twisted/conch/test/test_filetransfer.py

```diff
@@ -560,7 +562,9 @@ def test_openDirectoryIteratorDeprecated(self):
         d = self.client.openDirectory(b'')
         self._emptyBuffers()
         openDir = yield d
-        openDir.next()
+        d_ = openDir.next()
```

Comment:
can we have a better name for this variable, instead of just `d_`?

In the same time, I feel like the `_emptyBuffers` is abused here.

I think it should be made a public method, and maybe rename to "triggerCommunication" or something like that.

There is not point to make it private, as we are in testing code.

The only purpose of using a private method in a test is when you write something that should be use only by a single test